src/game/Battle.java

```diff
@@ -0,0 +1,30 @@
+package game;
+
+public class Battle {
+
+    private Ship ship1;
+    private Ship ship2;
```

Comment:
Dobra praktyka: jak sytuacja na to pozwala i nie przewidujesz setterów do danego pola, a jest ustawiane w jedynym dostępnym konstruktorze, dawaj je jako `final`.
`private final Ship ship1'